htdocs/admin/pdf.php

```diff
@@ -253,6 +256,33 @@
 	}
 }
 
+// Height of logo
+
+print '<tr class="oddeven"><td>'.$langs->trans("MAIN_DOCUMENTS_LOGO_HEIGHT").'</td><td>';
```

Comment:
Why moving the setup of the logo in main page into the section to setup adress frame ?

Reply:
@eldy it's done